editor/editor_node.cpp

```diff
@@ -7079,127 +6501,88 @@ EditorNode::EditorNode() {
 	title_bar = memnew(EditorTitleBar);
 	main_vbox->add_child(title_bar);
 
-	left_l_hsplit = memnew(HSplitContainer);
+	left_l_hsplit = memnew(DockSplitContainer);
+	left_l_hsplit->set_name("left_l_hsplit");
```

Comment:
Do we need these names for something or is it just for convenience? I don't mind having them in any case, but we should give them better names. Maybe at least something that indicates that they are related to docks. Like "DockHSplitLeftL", then "DockHSplitLeftR", etc. same for VSplits.

Reply:
I think I added it for debugging, so I could tell the difference between them, they aren't necessary.
They are convenient, so I'll keep and rename them.